app/views/arf_reports/_list.html.erb

```diff
@@ -21,11 +23,13 @@
                              :class => 'host_select_boxes',
                              :onclick => 'hostChecked(this)' %>
         </td>
-      <td><%= name_column(arf_report.host) %></td>
+      <td class="elipsis"><%= name_column(arf_report.host) %></td>
       <td><%= display_link_if_authorized(_("%s ago") % time_ago_in_words(arf_report.reported_at), hash_for_arf_report_path(:id => arf_report.id)) %></td>
       <td><%= report_arf_column(arf_report.passed, "label-info") %></th>
       <td><%= report_arf_column(arf_report.failed, "label-danger") %></th>
       <td><%= report_arf_column(arf_report.othered, "label-warning") %></th>
+      <td class="ellipsis"><%= display_link_if_authorized(arf_report.policy.name, hash_for_edit_policy_path(:id => arf_report.policy.id)) %></th>
+      <td class="ellipsis"><%= openscap_proxy_link arf_report %></th>
```

Comment:
I think it looks better if statistics columns are last. Could you move these two after report info?